src/background_gc.erl

```diff
@@ -74,7 +74,7 @@ interval_gc(State = #state{last_interval = LastInterval}) ->
     State#state{last_interval = Interval}.
 
 gc() ->
-    Enabled = rabbit_misc:get_env(rabbit, background_gc_enabled, true),
```

Comment:
`rabbit.background_gc_enabled` has an [application-level default](https://github.com/rabbitmq/rabbitmq-server/blob/master/Makefile#L117), so this should not break anything.